app/routes/board_routes.py

```diff
@@ -11,3 +11,17 @@ def get_all_boards():
     boards = Board.query.all()
     boards_response = [board.to_dict() for board in boards]
     return make_response(jsonify(boards_response), 200)
+
+@board_bp.route("", methods=["POST"])
+def create_board():
+    request_body = request.get_json()
+
+    if "title" not in request_body or "owner" not in request_body:
```

Comment:
You can use a try/except block with the from_dict class method that Alycia had set up in card.py. This reduce hard-coding in our program